browsertests/testutils_test.go

```diff
@@ -0,0 +1,216 @@
+package browser_test
```

Comment:
Maybe "package browsertests" to match the directory name?

Same for browser_test.go file.